packages/web-react/src/components/Dropdown/__tests__/Dropdown.test.tsx

```diff
@@ -5,57 +5,64 @@ import { classNamePrefixProviderTest } from '../../../../tests/providerTests/cla
 import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
 import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
 import Dropdown from '../Dropdown';
+import DropdownPopover from '../DropdownPopover';
+import DropdownTrigger from '../DropdownTrigger';
 
 describe('Dropdown', () => {
   classNamePrefixProviderTest(Dropdown, 'DropdownWrapper');
 
   stylePropsTest(
-    (props: Record<string, unknown>) => <Dropdown {...props} data-testid="test-dropdown" />,
+    (props: Record<string, unknown>) => (
+      <Dropdown id="dropdown" isOpen={false} onToggle={() => {}} {...props} data-testid="test-dropdown" />
+    ),
     'test-dropdown',
   );
 
-  restPropsTest(Dropdown, '.Dropdown');
+  restPropsTest(Dropdown, '.DropdownWrapper');
```

Comment:
Is there still a `DropdownWrapper` class?

Reply:
Maybe we should narrow this down in the future. What do you think @adamkudrna @crishpeen ?

Reply:
> yes we have `DropdownWrapper` for `<Dropdown />` and `Dropdown` for `<DropdownPopover />`

Sounds like we should fix the naming 🙂. In CSS but also in Figma.

Reply:
I will create an issue for Thursday's lengthy themes.